components/serial/loader.cpp

```diff
@@ -57,6 +65,12 @@ namespace Serial
 
     void Saver::save(const std::string& val) { mStream.write(val); }
 
+    void Saver::save(const Misc::Point& point)
```

Comment:
I would prefer not to do this here, and just add save() and load() methods to the Point class.
Otherwise, looks good to me :)